ovos_core/intent_services/service.py

```
@@ -180,19 +203,17 @@ def disambiguate_lang(message):
         """
         default_lang = get_message_lang(message)
         valid_langs = message.context.get("valid_langs") or get_valid_languages()
-        valid_langs = [standardize_lang_tag(l) for l in valid_langs]
+        valid_langs = [standardize_lang(l) for l in valid_langs]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)